### ISSUE TITLE
Will FRITZ!OS 7.29 and the FRITZ!Box 4060 be supported?

### ISSUE THREAD

**Issue by albi** [seydx/homebridge-fritz-platform#273]
Currently I can't enable the guest WiFi. The switch is displayed but without working.

**Comment by albi** [comment]
The LED Switch is working

**Comment by papa3333** [comment]
I can enable the guest-WLAN (FB 4060, 3 repater 6000, one Repeater 1750e for them all in Mesh

**Comment by albi** [comment]
@papa3333 sorry for the late reply, was on vacation. How did you manage it to switch on guest WiFi via HomeKit? I am still struggling with it. Have the same setup, FritzBox 4060 and one Repeater 6000.

**Comment by papa3333** [comment]
Herr some Screenshots from my homebridge.
If you need more help, please mail.

Kind regards
Werner



Von meinem iPhone gesendet

**Comment by albi** [comment]
@papa3333 Hi Werner, can't see any screenshots. :(

**Comment by papa3333** [comment]
2 nd try:



Von meinem iPhone gesendet

**Comment by albi** [comment]
nope. did not work

**Comment by papa3333** [comment]
![0C4B8D93-9E25-4ECC-97E4-F0967C9C6934](https://user-images.githubusercontent.com/32838747/174641724-0ffc38b2-89a9-4d7c-8404-aad00630e8f9.png)
![1E9706E2-9F13-4595-8C38-40E1606033FA](https://user-images.githubusercontent.com/32838747/174641736-c80241d2-e68e-40ec-91c1-28827126ad80.png)
![C360692B-4D4E-4009-9FC7-8371B7054A5A](https://user-images.githubusercontent.com/32838747/174641741-703f59c0-ae23-4b08-be14-669a88c2e095.png)
![33615AD6-8484-45B5-A75E-194620BE51EB](https://user-images.githubusercontent.com/32838747/174641744-a36031d3-a635-456d-9fc8-592703904398.png)
![44E52DF7-42B2-4B38-B77A-1BB90150F43F](https://user-images.githubusercontent.com/32838747/174641747-d3d495c6-9018-4028-b5be-f520fc12ef68.png)

PS: FritzOs 7.30 World too

**Comment by albi** [comment]
Hi Werner, 

danke für die Bilder. Habe die Konfiguration übernommen. Aber am Ende vom Tag zeigt es mir einfach nur das Gäste-Wifi als Schalter an, jedoch schaltet dieser nicht das Gäste-Wifi an oder aus. Kannst du das mal bei Dir testen ob es wirklich an-/ausgeschaltet wird?

Danke und Gruss
Alex

**Comment by papa3333** [comment]
Hab ich probiert. Gäste WIFI ist tatsächlich ein Schalter. Beim Einschalten des Gäste WIFI am 4060 werden alle im Mesh automatisch mit eingeschaltet. Gäste Netz ist sicht- und nutzbar. Habe die Abfrage von Benutzer/Passwort weggelassen.
Viele Grüße 
Werner

Von meinem iPhone gesendet

**Comment by albi** [comment]
Ah ok, du hast also kein username und passwort in der Konfiguration? Das wäre noch ein Test den ich machen könnte.

**Comment by papa3333** [comment]
Doch, hab ich. Ist aber nichts für die Öffentlichkeit.
Hast Du die FRITZ!Box auf Fernsteuerung geschaltet?

Von meinem iPhone gesendet

**Comment by albi** [comment]
Wo genau stell ich das ein? Ich meinte aber ja das habe ich.

doch gerade nachgeschaut der user hat „zugang auch aus dem internet"

**Comment by papa3333** [comment]
War ein Irrtum, meinte den Callmonitor. Aber der hat damit nichts zu tun

Von meinem iPhone gesendet

**Comment by papa3333** [comment]
Hat Deine FB Benutzername UND Passwort? Auch für das Web Interface? 

Von meinem iPhone gesendet

**Comment by albi** [comment]
im webinterface (lokal) reicht passwort. respektive ist das der automatisch angelegte user. habe aber noch einen zusätzlichen user, auch mit allen berechtigungen, bestehend aus username und passwort. diesen habe ich im plugin eingetragen.

versuchs grad mit dem von der fritzbox angelegten user. vllt. hat der doch mehr rechte.

**Comment by albi** [comment]
nein, das hat auch nichts gebracht. der schalter in homekit geht an und aus. aber er verändert nichts an der fritzbox.

**Comment by papa3333** [comment]
Habe den Standard User verändert, um alle Rechte zu erhalten. Ändert sich der Zustand des Repeaters automatisch mit?

Von meinem iPhone gesendet

**Comment by albi** [comment]
@seydx still not working. the switch for wifi_guest is visible. i can switch it on and off, but nothing happens on the fritzbox. the guest wifi is still off. checked username and password twice, and tried papa3333s configuration.

**Comment by albi** [comment]
naja ich hab das passwort und den username vom standarduser schon getestet mit dem plugin, auch das bringt nichts. gebe ich einen falschen user oder so ein, dann kann ich den switch in homekit bedienen, jedoch springt er nur kurz auf „on“ um dann wieder auf „off“ zurückzuspringen.

**Comment by papa3333** [comment]
Homebridge neu gestartet?

Von meinem iPhone gesendet

**Comment by albi** [comment]
Hab alles versucht. Irgendwie geht da nichts. Der Schalter ist sichtbar aber ohne Funktion.

@seydx still not working. tried different configs and users.

**Comment by albi** [comment]
also der callmonitor schaltet sich ein und aus? das gäste wifi nun auch?

**Comment by albi** [comment]
schon zig mal ;-)
